src/components/Link.react.js

```diff
@@ -71,7 +73,9 @@ export default class Link extends Component {
                 href={href}
                 onClick={e => this.updateLocation(e)}
             >
-                {this.props.children}
+                {isNil(this.props.children)
+                    ? this.props.href
+                    : this.props.children}
```

Comment:
While we're golfing - add `children` to the destructuring of `this.props` above (`href` is already there) and this collapses to just `{isNil(children) ? href : children}`

That's a nicer pattern anyway, as it shows up front what props get used during the render.

Reply:
Makes sense, destructuring does look cleaner. Updated in 065c462.